exercises/binary-search/instructions.md

```diff
@@ -0,0 +1,25 @@
+# Instructions
+
+Your task is to implement a binary search algorithm.
+
+A binary search algorithm sorts a list by repeatedly splitting it in half, only keeping the half which contains the item we're looking for.
+It allows us to quickly narrow down the possible locations of our item until we find it, or until we've eliminated all possible locations.
+It only works when a list has been sorted.
```

Comment:
I wonder if this would be nice to put into an `exercism/caution` block, as it is very important.

Reply:
You mean just this last sentence?

Reply:
Yeah. Sorry about not being specific.

Reply:
Cool thanks, yeah, I will do that.